tss-esapi/src/structures/buffers.rs

```diff
@@ -106,7 +106,7 @@ pub mod sensitive_data {
 }
 
 pub mod private {
-    buffer_type!(Private, 256, TPM2B_PRIVATE);
+    buffer_type!(Private, 1550, TPM2B_PRIVATE);
```

Comment:
Where does 1550 derive from? Should this be defined as a named constant with some description? To an outside observer like me, this looks like an arbitrary number and I can't see why the number has changed when the structure hasn't.

Reply:
That's true, I should probably use `::std::mem::size_of::<_PRIVATE>()`. The reason is that the spec isn't exactly clear about that size, it just says it must be at most essentially `::std::mem::size_of::<_PRIVATE>()`, with some definition given for `_PRIVATE`. Will change.